Analyzer/src/MakeNNVariables.cc

```diff
@@ -166,145 +179,140 @@ void MakeNNVariables::Loop(NTupleReader& tr, double, int maxevents, bool)
+                            "dPhi_Stop1Stop2_cm_OldSeed"+myVarSuffix,
+                            "Stop1_mass_cm_OldSeed"+myVarSuffix,     "Stop2_mass_cm_OldSeed"+myVarSuffix,
+                            "Stop1_pt_cm_OldSeed"+myVarSuffix,       "Stop2_pt_cm_OldSeed"+myVarSuffix,
+                            "Stop1_phi_cm_OldSeed"+myVarSuffix,      "Stop2_phi_cm_OldSeed"+myVarSuffix,
+                            "Stop1_eta_cm_OldSeed"+myVarSuffix,      "Stop2_eta_cm_OldSeed"+myVarSuffix,
+                            "Stop1_scalarPt_cm_OldSeed"+myVarSuffix, "Stop2_scalarPt_cm_OldSeed"+myVarSuffix,
+                        };
+
+                        std::set<std::string> varTopSeed =
+                        {
+                            "dR_Stop1Stop2_cm_TopSeed"+myVarSuffix,
+                            "dPhi_Stop1Stop2_cm_TopSeed"+myVarSuffix,
+                            "Stop1_mass_cm_TopSeed"+myVarSuffix,     "Stop2_mass_cm_TopSeed"+myVarSuffix,
+                            "Stop1_pt_cm_TopSeed"+myVarSuffix,       "Stop2_pt_cm_TopSeed"+myVarSuffix,
+                            "Stop1_phi_cm_TopSeed"+myVarSuffix,      "Stop2_phi_cm_TopSeed"+myVarSuffix,
+                            "Stop1_eta_cm_TopSeed"+myVarSuffix,      "Stop2_eta_cm_TopSeed"+myVarSuffix,
+                            "Stop1_scalarPt_cm_TopSeed"+myVarSuffix, "Stop2_scalarPt_cm_TopSeed"+myVarSuffix,
+                        };
+
+                        std::set<std::string> varTops = 
+                        {
+                            "top1_pt_cm"+myVarSuffix,   "top2_pt_cm"+myVarSuffix,
+                            "top1_eta_cm"+myVarSuffix,  "top2_eta_cm"+myVarSuffix,
+                            "top1_phi_cm"+myVarSuffix,  "top2_phi_cm"+myVarSuffix,
+                            "top1_mass_cm"+myVarSuffix, "top2_mass_cm"+myVarSuffix,
+                        };
 
-                std::set<std::string> var7toLastJet =
-                {
-                    "combined7thToLastJet_pt_cm"+myVarSuffix,
-                    "combined7thToLastJet_eta_cm"+myVarSuffix,
-                    "combined7thToLastJet_phi_cm"+myVarSuffix,
-                    "combined7thToLastJet_m_cm"+myVarSuffix,
-                    "combined7thToLastJet_E_cm"+myVarSuffix,
-                };
-
-                // -----------------------------------------------
-                // get the jet variables separately for 0l, 1l, 2l
-                // -----------------------------------------------
-
-                std::string ptCut = "pt30";
-                
-                std::set<std::string> varChannelSpecific =
-                {
-                    "HT_trigger_"+ptCut+myVarSuffix,
-                    "NGoodJets_"+ptCut+"_double"+myVarSuffix,
-                };
+                        std::set<std::string> var7toLastJet =
+                        {
+                            "combined7thToLastJet_pt_cm"+myVarSuffix,
```

Comment:
Is this the one we want for all channels ? I figured we would also need a 6thToLast and 8thToLast